docs/sql-migration-guide.md

```diff
@@ -83,7 +83,9 @@ license: |
 
   - In Spark 3.2, `TRANSFORM` operator can support `ArrayType/MapType/StructType` without Hive SerDe, in this mode, we use `StructsToJosn` to convert `ArrayType/MapType/StructType` column to `STRING` and use `JsonToStructs` to parse `STRING` to `ArrayType/MapType/StructType`. In Spark 3.1, Spark just support case `ArrayType/MapType/StructType` column as `STRING` but can't support parse `STRING` to `ArrayType/MapType/StructType` output columns.
 
-  - In Spark 3.2, the unit-to-unit interval literals like `INTERVAL '1-1' YEAR TO MONTH` are converted to ANSI interval types: `YearMonthIntervalType` or `DayTimeIntervalType`. In Spark 3.1 and earlier, such interval literals are converted to `CalendarIntervalType`. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.interval.enabled` to `true`.
+  - In Spark 3.2, the unit-to-unit interval literals like `INTERVAL '1-1' YEAR TO MONTH` and the unit list interval literals like `INTERVAL '3' DAYS '1' HOUR` are converted to ANSI interval types: `YearMonthIntervalType` or `DayTimeIntervalType`. In Spark 3.1 and earlier, such interval literals are converted to `CalendarIntervalType`. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.interval.enabled` to `true`.
+
+  - In Spark 3.2, the unit list interval literals support year-month units (YEAR and MONTH), day-time units (DAY, HOUR, MINUTE and SECOND) and the others (WEEK, MILLISECOND and MICROSECOND). Within an interval literal, multiple units are allowed to be listed as long as all of the units belong to the same category like `5 YEARS 8 MONTHS` or `1 DAY 2 MINUTE`. In Spark 3.1 and earlier, any units are allowed to be listed within a literal. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.interval.enabled` to `true`.
```

Comment:
let's highlight the breaking change:
```
In Spark 3.2, the unit list interval literals can not mix year-month fields (YEAR and MONTH) and
day-time fields (WEEK, DAY, ..., MICROSECOND). For example, `INTERVAL 1 day 1 hour` is invalid
in Spark 3.2. In Spark 3.1 and earlier, there is no such limitation and the literal returns value of
`CalendarIntervalType`. To restore ...
```

Reply:
Thank you for the suggestion.
BTW, `WEEK`, `MILLISECOND` and `MICROSECOND` are not valid ANSI interval units are they?

Reply:
they are not, but they are valid day-time fields.